demos/pycram_multiverse_demo/testing_giskard.py

```diff
@@ -0,0 +1,144 @@
+import actionlib
+from control_msgs.msg import GripperCommandGoal, GripperCommandAction
+from giskard_msgs.msg import WorldResult
+
+from giskardpy.python_interface.old_python_interface import OldGiskardWrapper as GiskardWrapper
+import rospy
```

Comment:
The problem was that currently importing from ros folders of pycram will initialize a node in ros called "pycram", and I didn't want that, but ok I don't think it will cause problems now, so I will change that.